core/modules/link/link.module

```diff
@@ -252,6 +252,23 @@ function link_field_settings_form_validate($element, &$form_state, $complete_for
   }
 }
 
+/**
+ * #element_validate handler for custom form usage.
+ */
+function link_field_custom_form_validate($element, &$form_state, $form) {
```

Comment:
Is there a reason to include this function in the module file, given that it's only used by the test?

Reply:
Initially I was going to leave validation out of this, but @docwilmot suggested it would be good to include some validation.

Reply:
Closing as outdated.